forte/data/ontology/top.py

```diff
@@ -1189,42 +1250,30 @@ class BoundingBox(Box):
     the image/grid.
 
     Args:
-        pack: The container that this BoundingBox will
-            be added to.
+        pack: the container that this ``Box`` will be added to.
+        tl_point: the indices of top left point of the box, the unit is one
+            pixel.
+        br_point: the indices of bottom right point of the box, the unit is one
+            pixel.
         image_payload_idx: the index of the image payload in the DataPack's
             image payload list. If it's not set,
-            it defaults to 0 which means it will load the first image payload.
-        height: the height of the bounding box, the unit is one image array
-            entry.
-        width: the width of the bounding box, the unit is one image array entry.
-        grid_height: the height of the associated grid, the unit is one grid
-            cell.
-        grid_width: the width of the associated grid, the unit is one grid
-            cell.
-        grid_cell_h_idx: the height index of the associated grid cell in
-            the grid, the unit is one grid cell.
-        grid_cell_w_idx: the width index of the associated grid cell in
-            the grid, the unit is one grid cell.
-
+            it defaults to 0 which meaning it will load the first image payload.
     """
 
-    def __init__(
-        self,
+    @classmethod
+    def init_from_center_n_shape(
```

Comment:
What's the difference between `BoundingBox.init_from_center_n_shape` and `Box.init_from_center_n_shape`? If they are exactly the same, then why do we need `BoundingBox` for this PR?

Reply:
I haven't thought of an actually difference between `BoundingBox` and `Box` yet. Previously, `BoundingBox` requires `Grid` to function correctly but it doesn't seem to be the case. I am thinking we can keep it here and we can add more attributes/functions as needed later.

Reply:
Removed and it's ready for review now.